staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -946,6 +946,10 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// Sysctls is a white list of allowed sysctls in a pod spec. Each entry
+	// is either a plain sysctl name or ends in "*" in which case it is considered
+	// as a prefix of allowed sysctls.
+	Sysctls []string `json:"sysctls,omitempty"`
```

Comment:
proto tags are missing

Reply:
they got added automatically in the generation commit, but having them in this commit makes them easier to review

Reply:
Moved from the generated code.